sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -100,7 +99,7 @@ class CosmosClient(object):  # pylint: disable=client-accepts-api-version-keywor
     :keyword str consistency_level: Consistency level to use for the session. Default value is None (account-level).
         More on consistency levels and possible values: https://aka.ms/cosmos-consistency-levels
     :keyword int timeout: An absolute timeout in seconds, for the combined HTTP request and response processing.
-    :keyword int request_timeout: The HTTP request timeout in milliseconds.
+    :keyword int connection_timeout: The HTTP request timeout in milliseconds.
```

Comment:
connection_timeout shoud be in seconds.
I think this was why we had the `/1000.0` in the code. We probably need to keep the milliseconds unit if `request_timeout` is used, otherwise seconds if `connection_timeout` is used....